### ISSUE TITLE
Flow Run Time Filtering 

### ISSUE THREAD

**Issue by mattconger** [PrefectHQ/prefect#8582]
### First check

- [X] I added a descriptive title to this issue.
- [X] I used the GitHub search to find a similar request and didn't find it.
- [X] I searched the Prefect documentation for this feature.

### Prefect Version

2.x

### Describe the current behavior

There currently is no way to filter flow runs or task runs in the UI.  It is possible to see the different length of time for a few flow runs using the timeline view; however, when the of subflow runs or tasks runs enters the hundreds, this becomes too cluttered to view using the timeline chart.  

### Describe the proposed behavior

Have the ability to sort flow and task runs by duration.  Could possibly be implemented using either using total_run_time or estimated_run_time. 

### Example Use

The user has a few hundred subflows that are kicked off by a parent flow.  The user wants to be able to sort the subflows in order of duration to quickly sort the subflows and see which ones took the longest.

**Comment by zhen0** [comment]
@madkinsz - can I get your help with triaging this one? The UI side for this is small-ish but but for it to work well, we'd ideally add a sort to the api for `run_time`. Can you give an idea of priority from a python side? Could we add to the team orchestration backlog?

**Comment by madkinsz** [comment]
I'm not sure how complicated these are to sort by given that they are computed fields.

**Comment by billpalombi** [comment]
I agree this would be an improvement to the flow runs page. We'll probably iterate on the UI in the future, but having this sort ability would be valuable regardless. I'm going to edit the description a bit to clarify the intent.
